browser/src/Editor/NeovimEditor/NeovimEditor.tsx

```diff
@@ -727,6 +741,31 @@ export class NeovimEditor extends Editor implements IEditor {
         this._scheduleRender()
     }
 
+    public async getVimHighlights(): Promise<IVimHighlight[]> {
+        const configurationColors: Array<{
+            settings: string
+            scope: string
+        }> = this._configuration.getValue("editor.tokenColors")
+
+        // FIXME: Specify Highlights to check by differentiating vim highlights as no having "."
+        const tokens = Object.keys(configurationColors).filter(c => !c.includes("."))
```

Comment:
Could we move this logic to `NeovimInstance` as `getHighlights`? This way, we don't need a public `getVimHighlights` method, as I'm hoping to keep the `Editor` interface as agnostic of Vim as possible.

Once we have that, we could change below to:
```
const vimHighlights = await this._neovimInstance.getHighlights()
```